sub-packages/bionemo-geneformer/tests/bionemo/geneformer/scripts/test_train_geneformer.py

```diff
@@ -29,15 +29,21 @@
 from bionemo.testing import megatron_parallel_state_utils
```

Comment:
It would be helpful to add a test case with bypass_tokenizer_vocab=False and ensuring that the correct error is raised.